### PR TITLE
add dashboard status indicator

### PR DESCRIPTION
**Pull request by yjbanov**
screen shot 2016-05-27 at 3 38 57 pm

/cc @devoncarew

### CODE REVIEW

**Comment by yjbanov** [comment]
I'll TBR it. I'd like to get it in soon enough to be able to test it on the actual box.

File: web/index.html [hunk 1]
```diff
@@ -37,27 +37,29 @@
...
-            </svg>
+        <div class="metric-data build-status-card">
+          <svg title="Linux" id="buildbot-linux-status" version="1.0" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" viewBox="0 0 266 312" width="55px" height="75px">
+           <g transform="translate(-3.3359375,285.2793)">
```
Comment (devoncarew) [inline web/index.html]: we may want to split these svgs out into separate files at some point

**Comment by devoncarew** [comment]
lgtm

File: web/index.html (hunk 1 above)
Comment (yjbanov) [inline web/index.html]: Yeah, it would certainly help with viewing diffs.

**Pull request merged**